src/main/java/net/revelc/code/formatter/xml/lib/XmlDocumentFormatter.java

```diff
@@ -82,8 +83,12 @@ private void copyNode(Reader reader, FormatState state) throws IOException {
             state.out.append(new XMLTagFormatter().format(tag.getTagText(), indentBuilder.toString(),
                     fDefaultLineDelimiter, prefs));
         } else {
-            state.out.append(tag.getTagText());
-        }
+	    String tagText = tag.getTagText();
+	    if (!prefs.getDeleteBlankLines()
+		    || (prefs.getDeleteBlankLines() && tagText != null && !tagText.trim().equalsIgnoreCase(""))) {
```

Comment:
This library can probably move to Java 11, and use `isBlank`, but even if we're not going to do that yet, `isEmpty` is an improvement here:

```suggestion
            || (prefs.getDeleteBlankLines() && tagText != null && !tagText.trim().isEmpty())) {
```